paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/CompactAction.java

```diff
@@ -74,6 +78,13 @@ public CompactAction withPartitions(List<Map<String, String>> partitions) {
         return this;
     }
 
+    static Options getCatalogOptions(MultipleParameterTool params) {
+        Map<String, String> catalogConfig = getConfigMap(params, "catalog-conf");
```

Comment:
`getConfigMap` may return null, so we need to check nullability.
Can we do it like this in `create`:
```
Tuple3<String, String, String> tablePath = getTablePath(params); 
if (tablePath == null) {
    return Optional.empty();
}
```
And then this method is not necessary.